suite/refill/eth_refill_test.go

```diff
@@ -81,15 +81,25 @@ var _ = Describe("FluxAggregator ETH Refill", func() {
 		})
 
 		By("Adding FluxAggregator jobs to nodes", func() {
-			os := &client.PipelineSpec{
+			// create the bridge
+			bta := client.BridgeTypeAttributes{
+				Name:        "variable",
 				URL:         adapter.ClusterURL() + "/variable",
-				Method:      "GET",
 				RequestData: "{}",
-				DataPath:    "data,result",
+			}
+
+			os := &client.PipelineSpec{
+				BridgeTypeAttributes: bta,
+				DataPath:             "data,result",
 			}
 			ost, err := os.String()
 			Expect(err).ShouldNot(HaveOccurred())
+
 			for _, n := range nodes {
+				// create the brige
```

Comment:
Typo, but I always try to avoid comments for code that is pretty self explanatory. Comment is create the bridge, the function call below is `CreateBridge`.